contrib/kind-common.sh

```
@@ -735,42 +740,81 @@ delete_metallb_dir() {
+    return 0
+  fi
+
+  local timeout
+  timeout=$(calculate_timeout ${endtime})
+  echo "Waiting for k8s to launch all ${ds} pods (timeout ${timeout})..."
+  # `kubectl rollout status` errors on DaemonSets with updateStrategy=OnDelete
+  # (upgrade-ovn.sh sets ovs-node to OnDelete so helm upgrade doesn't roll
+  # OVS out from under still-running ovnkube-node pods). For OnDelete DSes
+  # we can't observe rollout progress; just wait for pods to be Ready.
+  local strategy
+  strategy=$(kubectl -n ovn-kubernetes get daemonset ${ds} \
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)